.github/workflows/gke-kube-test-command.yml

```diff
@@ -1,162 +0,0 @@
-name: GKE Kube Acceptance Test
```

Comment:
@supertopher @git-phu @cpdeethree as infra/tooling folks, I want your thoughts on whether we should delete/keep this.

This workflow was actually developed so we can test OSS kube changes on GKE since all cloud providers implement Kubernetes differently. We haven't run this in a while since we haven't worked on the Kubernetes layer in a while. Because of this, I'm not sure if this works well as is.

The pros of keeping this is we have something we can run tests against if we do need this. Since this contains alot of the boilerplate code to standing up and connecting to a GKE cluster, we might be able to adapt this to more interesting applications in the future e.g. deploying Airbyte to GKE and running a load test etc.

The cons of keeping this is.. since we don't use this often, there is a risk of this going out of date.

I want to crowd-source this decision since some of this is passing out of my hands to you guys, so I think 3 of you should be the final decision makers on this after being given the context.

Reply:
I personally lean towards keeping this since I see lots of potential applicaations especially now Conor has joined the team and we have more bandwidth. I want to index more heavily on you guys in this decision.

Reply:
Does it make sense to "archive" some of these workflows to mark them as not actively running and possibly needing work, but not deleting them entirely?

Reply:
I think we should keep this one and force it to run via cron at some interval.  Maybe once a week.  it will help us blame any breaks that occur.  I know that we aren't actively editing this part of the codebase, but I don't think we would remember to un-archive it when we do.

this also solves the stale runs problem in the short term.

Reply:
Added a cron schedule!